nuxt-frontend/assets/css/global.css

```diff
@@ -189,52 +189,58 @@
   }
 
   .header-1 {
-    @apply font-opensans-bold tracking-[-0.08rem] xl:tracking-[-0.15rem]  text-[2rem] leading-[2.6rem] md:text-[2.875rem]  md:leading-[3.688rem] xl:text-[3.75rem] xl:leading-[4.875rem];
+    @apply font-opensans-bold tracking-[-0.08rem] xl:tracking-[-0.15rem]  text-[2rem] leading-[2.6rem] lg:text-[3.75rem] lg:leading-[4.875rem];
```

Comment:
shouldn't use `text-2xl` and `text-6xl` from tailwind?

Reply:
by using text-2xl and text-6xl also we have to override line height so i thought this is fine or what you say

Reply:
as you have already set the custom line height by leading, text line height never works there. and when you haven't set line height by leading, it will take text line height